src/config/proposalTemplates/communitySpend.tsx

```diff
@@ -0,0 +1,75 @@
+import { useMemo, useRef } from "react";
```

Comment:
I'm not sure this file is needed - you're already adding the `CommunitySpend` section to `ProposalForm.tsx`. 

If it's still intended, probably best to remove the addition to `ProposalForm` as that's confusing. **_ProposalForm_** has been getting a little big, so not surprised to see it broken down.